src/og/control/EarthCoordinates.ts

```diff
@@ -159,45 +169,42 @@ class EarthCoordinates extends Control {
         if (this._centerMode) {
             this.renderer.div.appendChild(this._createCenterEl());
             this.renderer.activeCamera.events.on("moveend", this._grabCoordinates, this);
-            this.renderer.activeCamera.events.on("moveend", throttle((e) => this._showHeight(), 400, true), this);
+            this.renderer.activeCamera.events.on("moveend", throttle((e: any) => this._showHeight(), 400, true), this);
         } else {
             this.renderer.events.on("mousemove", this._grabCoordinates, this);
-            this.renderer.events.on("mousestop", throttle((e) => this._showHeight(), 400, true), this);
+            this.renderer.events.on("mousestop", throttle((e: any) => this._showHeight(), 400, true), this);
         }
 
         this._refreshCoordinates();
 
         this._updateUnits();
     }
 
-    _grabCoordinates(px) {
+    _grabCoordinates(px: any) {
         let scrPx;
         let r = this.renderer;
         if (this._centerMode) {
             scrPx = r.handler.getCenter();
         } else {
             scrPx = px;
         }
-        this._lonLat = this.planet.getLonLatFromPixelTerrain(scrPx);
+        this._lonLat = this.planet?.getLonLatFromPixelTerrain(scrPx) as LonLat;
```

Comment:
`as LonLat` is redundant if `this._lonLat: LonLat` and extract `if(this._planet)`, or use `this._planet!.get..` it's fine.